src/components/layout/Section.jsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import cx from 'classnames';
+import PropTypes from 'prop-types';
+
+export const VALID_BG_NORMAL = ['white', 'haze', 'lightGray'];
+
+export const VALID_BG_INVERTED = ['navy'];
+
+export const VALID_PADDING = ['default', 'double', 'half', 'none'];
+
+/**
+ * @param {String} axis 'h' or 'v'
+ * @param {String} amount
+ * @returns {String} padding utility classes
+ */
+export const getPaddingClasses = (axis, amount) => {
+  const sides = axis === 'h' ? ['left', 'right'] : ['top', 'bottom'];
```

Comment:
@rey-wright any strong feelings about `h`/`v` versus `x`/`y`?

Reply:
No strong strong feelings... but I do think x & y as concepts are a lot more apparent what they're doing. Like if you saw just x & y in a vacuum, and asked them what these things meant, they might say "oh it represents an axis or a chart) or something like that, whereas `h` and `v` themselves aren't as apparent. I will say that it's slightly weird that we're spelling out each one but these (though not too weird).

So yeah personal preference in order, would be to use x/y, then spell them out with the full name, and then use h/v... but not super strong feelings any way.

Reply:
agreed, going to go with `x`/`y`; those names sit better with me because at least they're more familiar